### ISSUE TITLE
'AccessorSignature': is not a member of 'v8' 

### ISSUE THREAD

**Issue by rushangs29** [nospaceships/node-raw-socket#83]
Getting the error like this while updating a version to 20.9.0 from 18.16.0 : 

\node_modules\raw-socket\node_modules\nan\nan_callbacks.h(55,23): error C2039: 'AccessorSignature': is not a member of 'v8' [\node_modules\raw-socket\build\raw.vcxproj]
npm ERR!   (compiling source file '../src/raw.cc')
npm ERR!   \AppData\Local\node-gyp\Cache\20.9.0\include\node\v8.h(71,11):
npm ERR!   see declaration of 'v8'

**Comment by psznm** [comment]
For anyone looking for workaround, overriding version of "nan" package that raw-socket depends on, to version 2.17.0 worked for me. That is done by adding this to package.json

```
  "overrides": {
    "nan@2.14.*": "2.17.0"
  },
```

**Comment by Blog404DE** [comment]
hmm, thx - i've testet your device and i still got the same error message.

My package.json look like:

```json
{
    "name": "__projectname__",
    "description": "__projectdescription__",
    "version": "1.5.0",
    "author": "__my <email@address.tld>",
    "license": "UNLICENSED",
    "private": true,
    "repository": {
        "type": "git",
        "url": "https://__mygitrepository__.git"
    },
    "overrides": {
        "nan@2.14.*": "2.18.0"
    },
    "dependencies": {
        "@pm2/io": "^5.0.0",
        "console-stamp": "^3.1.1",
        "is-valid-domain": "0.1.6",
        "maxmind": "^4.3.11",
        "net-ping": "^1.2.3",
        "node-yaml-config": "1.0.0",
        "nodejs-traceroute": "^2.0.1",
        "posix": "^4.2.0",
        "raw-socket": "^1.7.0",
        "ws": "^8.13.0",
        "ws-rate-limit": "0.0.0"
    }
```

But you where on the right track: i've installed the source code from nan 2.18 "by hand" in node_modules and now, raw-socket is compiling without any problem. Of course, this is no real solution and not recommend in any way.  

Maybe i've clone the original raw-socket repository, change package.json and use my repository of raw-socket.

**Comment by stephenwvickers** [comment]
Fixed in raw-socket 1.8.1
